features/step_definitions/wait_steps.rb

```diff
@@ -53,30 +59,57 @@
 end
 
 When('I wait for the section element that takes a while to disappear') do
-  @test_site.section_experiments.removing_parent_section.wait_for_no_removing_section_element
+  @test_site
```

Comment:
lets put this onto 2 lines. `@test_site.section_experiments` then chain the other 2 on line 2